extensions_admin/pulp_rpm/extensions/admin/copy_commands.py

```diff
@@ -80,7 +80,8 @@ def generate_override_config(self, **kwargs):
 
         if kwargs[FLAG_RECURSIVE.keyword]:
             override_config[CONFIG_RECURSIVE] = True
-        elif kwargs.get(FLAG_RECURSIVE_CONSERVATIVE.keyword):
+
```

Comment:
My line of thought here is that --recursive-conservative should work even if you also specify --recursive.